products/opensuse/main.pm

```diff
@@ -370,6 +370,31 @@ else {
             loadtest "network/wireguard";
             return 1;
         }
+        if (get_var('OVS')) {
+            set_var('INSTALLONLY', 1);
+            if (check_var('HOSTNAME', 'server')) {
+                barrier_create('ipsec_done',          2);
+                barrier_create('traffic_check_done',  2);
+                barrier_create('password_ok',         2);
+                barrier_create('ipsec1_done',         2);
+                barrier_create('traffic_check_done1', 2);
+                barrier_create('ipsec2_done',         2);
+                barrier_create('traffic_check_done2', 2);
+                barrier_create('cert_done',           2);
+                barrier_create('host2_cert_ready',    2);
+                barrier_create('cacert_done',         2);
+            }
+            loadtest 'installation/bootloader_start';
+            loadtest 'network/setup_multimachine';
+            if (check_var('HOSTNAME', 'server')) {
+                loadtest 'console/ovs_server';
+            }
+            else {
+                loadtest 'console/ovs_client';
+            }
+            return 1;
```

Comment:
You don't need `return 1;` as you don't need it in `sle/main.pm`

Reply:
If i don't use it here, it will schedule one more pm that I don't need (the `system_prepare`). It has to do again with the different structures of the main.pm in sle and opensuse, so I considered it as a quick way of solving it.

Reply:
Ah ye, sorry I didn't think that there are this differences :open_mouth: